### PR TITLE
Dce asserts

### PR DESCRIPTION
**Pull request by erwinmombay**
Fixes https://github.com/ampproject/amphtml/issues/4905

### CODE REVIEW

**Comment by erwinmombay** [comment]
removed all deprecated tests as well

**Comment by erwinmombay** [comment]
@cramforce ready for review PTAL

File: build-system/runner/src/org/ampproject/AmpCommandLineRunner.java [hunk 1]
```diff
@@ -46,8 +47,10 @@
   /**
    * List of string suffixes to eliminate from the AST.
    */
-  ImmutableSet<String> suffixTypes = ImmutableSet.of(
-      "dev.fine");
+  ImmutableMap<String, Set<String>> suffixTypes = ImmutableMap.of(
+      "module$src$log.dev",
+          ImmutableSet.of("assert", "fine", "assertElement", "assertString", "assertNumber"),
```
Comment (erwinmombay) [inline build-system/runner/src/org/ampproject/AmpCommandLineRunner.java]: @cramforce please review list of method names

Comment (cramforce) [inline build-system/runner/src/org/ampproject/AmpCommandLineRunner.java]: Should also do `assertBoolean`. Not all are implemented yet :)

Comment (erwinmombay) [inline build-system/runner/src/org/ampproject/AmpCommandLineRunner.java]: added

**Comment by erwinmombay** [comment]
@cramforce anything else i need to follow up with here?

**Comment by cramforce** [comment]
LGTM

**Pull request merged**